info/v1/machine.go

```diff
@@ -167,6 +167,13 @@ type MachineInfo struct {
 	// The amount of memory (in bytes) in this machine
 	MemoryCapacity uint64 `json:"memory_capacity"`
 
+	// Number of memory DIMMs in this machine.
+	// 0 if number is uknown (e.g. in virtualized envirtonment).
+	MemoryDimmCount uint `json:"memory_dimm_count"`
```

Comment:
Can this be inferred from the DimmCount in MemoryByType below?

Reply:
Sure, it can. If you want I can remove it but in my opinion cost of calculating it and sending it too a client is pretty low while we still expose general information about all the DIMMs. It's your call :)

Reply:
I would prefer to keep the API as minimal as possible.